ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent-nonprod.json

```diff
@@ -40,5 +40,26 @@
     "CaseEventID": "placement",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "messageJudgeOrLegalAdviser",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
```

Comment:
Don't need to [give update permissions](https://tools.hmcts.net/confluence/display/RCCD/Using+CRUD+to+hide+Events+triggering+from+UI)